viahtml/blocklist.py

```diff
@@ -54,7 +54,6 @@ def is_blocked(self, url):
         self._refresh()
 
         domain = self._domain(url)
-        self.LOG.debug("DOMAIN? %s: %s", domain, self.domains)
```

Comment:
This was for debugging only. It's unrelated, but I can't be bothered to make a different PR for this